src/main/kotlin/org/rust/cargo/project/configurable/CargoConfigurable.kt

```diff
@@ -31,6 +37,24 @@ class CargoConfigurable(project: Project) : RsConfigurableBase(project) {
 
     private lateinit var cargoCheckArguments: CargoCommandLineEditor
 
+    private val cargoFeaturesComboBox = ComboBox<FeaturesSetting>().apply {
```

Comment:
Hm, I'm not sure that project settings are a good place for such properties. When you have several separate cargo projects, you may want to have a separate feature set for each cargo project
I think it's ok as initial stage but eventually, we should provide per cargo project settings
What do you think about it?

Reply:
Yeah, i did not think about the fact that we can attach more than 1 cargo project.
This makes the `additional features` field kind of unuseable when using multiple projects as passing a non-existing feature causes an error.

Do you have an idea where we could put a per Cargo project setting?

Reply:
@kumbayo currently, I've come up with three solutions:
* open some dialog on some action (for example, double click) on cargo project node in cargo tool window. But looks like it's absolutely undiscoverable
* add special action to cargo tool window to open a dialog where we can show panel with all cargo projects and the corresponding feature settings for each cargo project
* add panel from the point above to cargo settings (`Preferences | Languages & Frameworks | Rust | Cargo`)

I'd prefer the second variant